README.md

```diff
@@ -1 +1,154 @@
-# Metrics-webapp
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Stock Screaner <a name="about-project"></a>
+
+**Stock Screaner** is a tool quickly find current stock prices in USA companies...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Build With</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://redux-toolkit.js.org/">Redux toolkit</a></li>
+    <li><a href="https://site.financialmodelingprep.com/developer/docs/">Financial Modeling Prep API</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Filter companies by Company Names**
+- **Display all companies by Company Name and then filter by name or symbol**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ - [Live Demo](Coming soon)
```

Comment:
- [ ] As part of the project requirements, we are supposed to deploy and make the app accessible online. Let's work on it. I suggest using Netlify as it is easier and optimized for React apps. 🥸️

**Project requirements**
![Screenshot from 2023-03-22 19-18-10](https://user-images.githubusercontent.com/17243325/226973798-cf89644a-46dd-4739-aea4-40fe9c84cec8.png)

Reply:
thanks for your good and valuable feedback.

Reply:
resolved! ✔️